modules/integrate-circle-ci.adoc

```diff
@@ -0,0 +1,64 @@
+// Module included in the following assemblies:
+//
+// * integration/integrate-with-ci-systems.adoc
+:_module-type: PROCEDURE
+[id="integrate-circle-ci_{context}"]
+= Using CircleCI
+
+You can integrate {product-title} with CircleCI.
+
+.Prerequisetes
+* You must have a token with `read` and `write` permissions for the `Image` resource.
+* Useranme and password of your DockerHub account.
```

Comment:
For consistency, I'd start both with "You have a..."

Reply:
Also Useranme -> Username